cmake/packaging.cmake

```diff
@@ -71,11 +71,12 @@ endif()
 
 set (VS "") # VisualStudio
 if ( MSVC )
-    if     ( MSVC_VERSION STREQUAL 2100 )
+    # https://gitlab.kitware.com/cmake/cmake/blob/v3.10.0-rc5/Modules/Platform/Windows-MSVC.cmake#L51-68
```

Comment:
Thanks for updating the values!

I do not understand why this link is relevant for the code behind ... would it not be better to point to the official CMake documentation where this variable is documented?

https://cmake.org/cmake/help/latest/variable/MSVC_VERSION.html

Reply:
The link is to the Kitware code that generates the values of MSVC_VERSION.  They use these 1899, 1909, 1919 values which I've echoed into our code.

Reply:
I see ... but still, in my humble opinion, I think users reading this part of our CMake code would be more interested in reading the CMake documentation rather than the CMake code itself.